zookeeper-contrib/zookeeper-contrib-monitoring/check_zookeeper.py3

```diff
@@ -0,0 +1,404 @@
+#! /usr/bin/env python3
```

Comment:
`check_zookeeper.py3` should keep the same permission as check_zookeeper.py:
`-rwxr-xr-x@  1 maoling  staff  13057 Jun 20  2022 check_zookeeper.py*`
otherwise: permission denied: ./check_zookeeper.py3